Extension/src/js/content.js

```diff
@@ -1,9 +1,13 @@
+// function name should be like this => getFormURLEncoded
```

Comment:
Also function name should be something meaning full. getFormURLencoded doesn't convey what the function does.

Check this for all the variables that you have written 

a. they should be in camelCase
b. they should be meaningfull